### PR TITLE
Owlot/refactor with lib common

### PR DESCRIPTION
**Pull request by owlot**

### CODE REVIEW

**Review by amsmith-pro — CHANGES_REQUESTED** [review]
Looks great! Just one failing test

**Comment by owlot** [comment]
> Looks great! Just one failing test

fixed!

File: client-rest/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [package]
 description = "Arrow Storage service REST client"
```
Comment (amsmith-pro) [inline client-rest/Cargo.toml]: assets service

**Review by amsmith-pro — CHANGES_REQUESTED** [review]

File: client-rest/Cargo.toml (hunk 1 above)
Comment (owlot) [reply client-rest/Cargo.toml]: fixed!

**Review by amsmith-pro — APPROVED** [review]

**Pull request merged**